example-apps/kitchen-sink-new-router/source/components/form-controls/form2.ts

```diff
@@ -1,11 +1,9 @@
 import {Component} from '@angular/core';
 import {FORM_DIRECTIVES, Location} from '@angular/common';
-import {ROUTER_DIRECTIVES, RouterLink, RouteParams, Router}
```

Comment:
We forgot to remove this, the new example no longer has the old router as a dependency.